### ISSUE TITLE
Plugin / GUI feature wishlist

### ISSUE THREAD

**Issue by orsjb** [orsjb/HappyBrackets#28]
The IntelliJ setup instructions are still too hard to follow: setting up JDK and linking to project.
Improvements to plugin interface elements, e.g., a sync button, a pop-up menu on individual devices.
Ability to copy the name of the device.
A button that would pop up an SSH terminal logging into each device.
GUI method to copy sound and classes over to devices.
A button that takes the current sketch and makes it auto-run.

**Comment by angelofraietta** [comment]
Need to examine the singleton nature of plugin communication. A single version is running on from rats instance of tool window

**Comment by angelofraietta** [comment]
A button that would pop up an SSH terminal logging into each device. - This has been implemented as a copy ssh command to clipboard
